container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/label_value_consts/label_value_consts.go

```diff
@@ -15,10 +15,9 @@ const (
 	engineKurtosisResourceTypeLabelValueStr = "kurtosis-engine"
 	// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! DO NOT CHANGE THESE VALUES !!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-	enclaveKurtosisResourceTypeLabelValueStr            = "enclave"
-	apiContainerKurtosisResourceTypeLabelValueStr       = "api-container"
-	userServiceKurtosisResourceTypeLabelValueStr        = "user-service"
-	userServiceIngressKurtosisResourceTypeLabelValueStr = "user-service-ingress"
```

Comment:
The ingress like other user service resources uses the "user-service" resource type label.

Reply:
Fair enough, this will then follow the rules of retrieval/deletion of a user services as well?

Reply:
Yes because we want to collect all the user service resources.  We do the same for the K8S service and pod.